docs/interface.md

```diff
@@ -30,9 +30,9 @@ partial interface WorkerNavigator {
 };
 
 dictionary ShareData {
-  DOMString? title;
-  DOMString? text;
-  DOMString? url;
+  DOMString title;
+  DOMString text;
+  USVString url;
```

Comment:
It sounds like you have a reason to use USVString here, although the reason wouldn't be apparent from looking at the spec along. @annevk, WDYT, is there a reason to *not* use USVString in a situation like this?